src/components/Common/Username.tsx

```diff
@@ -75,7 +75,7 @@ const Username = ({ address, size, linked, variant = UsernameVariant.Full, stron
           {hasDclProfile && (
             <>
               <Avatar size={size} address={address} />
-              {profileHasName && !isAvatarVariant && <span>{profile!.name}</span>}
+              {profileHasName && !isAvatarVariant && <span className="dcl name">{profile!.name}</span>}
```

Comment:
If possible I'd name this `Username__Name` or something that follows our conventions